tests/acceptance/features/bootstrap/AppConfigurationContext.php

```diff
@@ -110,32 +111,31 @@ public function getAppParameter(string $capabilitiesApp, string $capabilitiesPat
 	 */
 	public function userRetrievesCapabilities(string $username): void {
 		$user = $this->featureContext->getActualUsername($username);
-		$this->userGetsCapabilities($user, true);
+		$response = $this->userGetsCapabilities($user, true);
+		$this->featureContext->setResponse($response);
```

Comment:
let's make this inline
```suggestion
		$this->featureContext->setResponse($this->userGetsCapabilities($user, true));
```